src/components/Form/Select.jsx

```diff
@@ -24,17 +24,23 @@ const StyledSelect = styled(VirtualizedSelect)`
   }
 
   && .Select-placeholder {
-    padding-left: 1.2rem;
-    line-height: 3.8rem;
+    padding-left: 1rem;
+    line-height: 2.3rem;
   }
 
   &&.Select--single {
     .Select-value {
-      padding-left: 1.2rem;
+      padding-left: 1rem;
     }
 
     .Select-value .Select-value-label {
-      line-height: 3.8rem;
+      line-height: 2.3rem;
+    }
+  }
+
+  &&.Select--multi {
```

Comment:
Isn't `&` enough (instead of 2) ?

Reply:
The selector weight is not higher enough with only one.

Reply:
... in that specific case